components/inputs/input-time-range.js

```diff
@@ -98,6 +109,30 @@ class InputTimeRange extends RtlMixin(LocalizeCoreElement(LitElement)) {
 		if (!this.label) {
 			console.warn('d2l-input-time-range component requires label text');
 		}
+
+		if (!this.startValue && !this.endValue) {
```

Comment:
It was necessary to give `startValue` and `endValue` default values depending on if the consumer sets them.

`startValue` should be midnight be default (used `getDefaultTime` from `input-time` in case that changes) and `endValue`, if it does not have a value, should be one time interval after (as seen in current time range usage in Brightspace).